runtime/query.go

```diff
@@ -65,7 +65,8 @@ func (r *Runtime) Query(ctx context.Context, instanceID string, query Query, pri
 	if err != nil {
 		return err
 	}
-	if inst.OLAPDriver == "druid" {
+	c, _, _ := r.OLAPDef(inst)
+	if c.Type == "druid" {
```

Comment:
`OLAPDef` feels too specific. Maybe do `handle = r.OLAP(instanceID)` and check `handle.Driver() == "druid"`?

Alternatively, it could have something like `r.ConnectorDef(instanceID, name)`?